src/index.ts

```diff
@@ -0,0 +1,10 @@
+export { IPeer } from "./interfaces/iPeer";
+
+// Debugging:
```

Comment:
Move this example's to the readme because commented code isn't allowed in the source